File: src/main/java/com/linebot/entity/BotLog.java
package com.linebot.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.Date;

@Data
@Builder
@Entity
@Table(name = "bot_log")
@NoArgsConstructor
@AllArgsConstructor
public class BotLog {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Integer id;
    @Column(name = "user_id")
    private String userId;
    @Column(name = "bot_action_name")
    private String botActionName;
    @Column(name = "message")
    private String message;
    @Column(name = "year_month")
    private String yearMonth;
    @Column(name = "day")
    private int day;
    @Column(name = "day_of_week")
    private int dayOfWeek;
    @Column(name = "hour")
    private int hour;
    @Column(name = "minute")
    private int minute;
    @Column(name = "exec_time")
    private Date execTime;

}

File: src/main/kotlin/com/linebot/entity/BotLog.kt
package com.linebot.entity

import java.util.*
import javax.persistence.*

@Entity
@Table(name = "bot_log")
data class BotLog(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        @Column(name = "id")
        var id: Int? = null,

        @Column(name = "user_id")
        var userId: String? = null,

        @Column(name = "bot_action_name")
        var botActionName: String? = null,

        @Column(name = "message")
        var message: String? = null,

        @Column(name = "year_month")
        var yearMonth: String? = null,

        @Column(name = "day")
        var day: Int = 0,

        @Column(name = "day_of_week")
        var dayOfWeek: Int = 0,

        @Column(name = "hour")
        var hour: Int = 0,

        @Column(name = "minute")
        var minute: Int = 0,

        @Column(name = "exec_time")
        var execTime: Date? = null
)
